Scheduling an Export: Described configuring export if remote server was selected. Updated entity types (#1605)
magento/merchdocs: src/system/data-schedule-export.md

   - For **Local Server**, enter a relative path within the Commerce installation, such as `var/export`. If the Remote storage module is configured, use `import_export/export`.
   - For **Remote FTP server**, enter the full URL and path to the target folder on the destination server.

> 1. If the **Remote FTP server** was selected enter connection credentials to the server and select additional settings:

Reviewer: ```suggestion
1. If the **Remote FTP **  server is selected, enter connection credentials to the server and select additional settings:
```

>    - **FTP Host[:Port]**, enter remote ftp host address.

Reviewer: ```suggestion
   - For **FTP Host[:Port]**, enter remote FTP host address.
```

>    - **User Name**, enter a username, to access the remote server.

Reviewer: ```suggestion
   - For **User Name**, enter the username used to access the remote server.
```

>    - **Password**, enter a password of provided username account.

Reviewer: ```suggestion
   - For **Password**, enter the password of the provided username account.
```

>    - **File Mode**, `Binary` or `ASCII`.

Reviewer: ```suggestion
   - **File Mode**, `Binary` or `ASCII`.
```
```suggestion
   - For **File Mode**, choose `Binary` or `ASCII`.
```

>    - **Passive Mode**, `No` or `Yes`.

Reviewer: ```suggestion
   - For **Passive Mode**, choose `No` or `Yes`.
```
